### DISCUSSION TITLE
Verbose Stitching Dir

### DISCUSSION THREAD

**Discussion by davidhardeniv-code** [OpenStitching/stitching#240] [Q&A]
I am trying to set the verbose_dir in my own script that I built off using the Sticher class but I don't want to change anything in the package scripts. Is there a way I can set this in my own script if anyone knows.

Thank you!

**Comment by lukasalexanderweber** [comment]
See https://github.com/OpenStitching/stitching/blob/a471af1a725d6525dcfe96436b27334a87bcd2bf/stitching/stitcher.py#L90

`stitcher.stitch_verbose(myimages, verbose_dir=mydirectory)`

**Reply by davidhardeniv-code** [reply]
Is there a way to do it without changing the sticher.py. I made a different script off of the stitcher.py so its not in my project. I would like to change it in my script without messing with the Stiitcher.py.

**Reply by lukasalexanderweber** [reply]
I can't help you without more information about your code

**Reply by davidhardeniv-code** [reply]
Below is the method I am using for the stitched batched in my own script

def stitch_batch(image_files, stitcher):
    return stitcher.stitch_verbose(image_files)

I tried this below but it didn't work.

def stitch_batch(image_files, stitcher):
    return stitcher.stitch_verbose(image_files, verbose_dir='file/path')

**Reply by lukasalexanderweber** [reply]
Is there an Error Message?

**Reply by davidhardeniv-code** [reply]
Yeah, below is the error

PROCESS ENDING EXCEPTION: Stitcher.stitch_verbose() got an unexpected keyword argument 'verbose_dir'
